linkatos/confirmation.py

```diff
@@ -0,0 +1,18 @@
+import linkatos.printer as printer
+from .utils import is_fresh_url
+
+
+def update_confirmation_if_url(parsed_message, expecting_confirmation):
+    if is_fresh_url(expecting_confirmation, parsed_message['type']):
+        expecting_confirmation = True
+
+    return (expecting_confirmation)
+
+
+def process_confirmation_if_yn(parsed_message, expecting_confirmation):
```

Comment:
It's harder for me to reason what is this function returning in what situation if you compose the resulting values in multiple places and the return in yet another place. As I said, it's a matter of style so your way is not wrong. Pick the one feels more readable to you.